phq/kafka/producer.py

```diff
@@ -6,12 +6,12 @@
 import confluent_kafka
 
 from .payload import pack_kafka_payload, Message
-from .settings import KAFKA_PRODUCER_BASE_CONFIGURATION
+from .settings import producer_base_configuration
 
 log = logging.getLogger(__name__)
 
 
-def _on_delivery_error_handler(err, message, ignore_large_message_errors=False):
+def _on_delivery_error_handler(err, ignore_large_message_errors=False):
```

Comment:
message is a required parameter of the function, so function definition should be:
`def _on_delivery_error_handler(err, message, ignore_large_message_errors=False):`